endpoints/openrtb2/auction.go

```diff
@@ -235,8 +246,38 @@ func (deps *endpointDeps) Auction(w http.ResponseWriter, r *http.Request, _ http
 		ao.Status = http.StatusInternalServerError
 		ao.Errors = append(ao.Errors, err)
 		return
+	} else if isRejectErr {
+		labels, ao = rejectAuctionRequest(*rejectErr, w, req.BidRequest, labels, ao)
+		return
+	}
```

Comment:
Can you update tests to cover these lines if possible?

Reply:
Actually, in this PR we do not invoke any hooks within the `exchange.HoldAuction` method, so there will be no rejection error and this code is not unreachable right now. But it will be added in next PR with corresponding tests, so if you think it is better to get out of this code from current PR - we can remove it for now.